sdk/java/src/main/java/sawtooth/sdk/client/Signing.java

```diff
@@ -1,3 +1,17 @@
+/* Copyright 2017 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+------------------------------------------------------------------------------*/
+
 package sawtooth.sdk.client.sawtooth.sdk.client;
```

Comment:
If you are going to move the directory, you need to update the package declaration, too.